specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -589,6 +644,47 @@
       },
       "x-ms-azure-resource": true
     },
+    "ResourceNameAvailabilityInput": {

```

Comment:
I think the name should be `CheckNameAvailabilityParameters`.

Reply:
Done in New PR: https://github.com/Azure/azure-rest-api-specs/pull/5496